app/components/Modal/DepositModal.jsx

```diff
@@ -170,16 +195,33 @@ class DepositModalContent extends DecimalChecker {
             );
         }
 
-        if (!!gatewayStatus[selectedGateway].simpleAssetGateway) {
+        if (
+            !!gatewayStatus[selectedGateway].simpleAssetGateway &&
+            !!backingAsset.gatewayWallet
+        ) {
+            let memoText;
```

Comment:
The if before did not check `!!backingAsset.gatewayWallet`. Backwards compatibility ensured?

Reply:
> The if before did not check `!!backingAsset.gatewayWallet`. Backwards compatibility ensured?

Yes, backward compatibility is ensured.